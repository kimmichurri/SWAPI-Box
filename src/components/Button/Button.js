import React from 'react';

const Button = ({ getPeople, getPlanets, getVehicles }) => {

        return (
            <div>
                <button onClick={getPeople} className="card-category-buttons people">people</button>
                <button onClick={getPlanets} className="card-category-buttons planets">planets</button>
                <button onClick={getVehicles} className="card-category-buttons vehicles">vehicles</button>
            </div>
        )
}

export default Button