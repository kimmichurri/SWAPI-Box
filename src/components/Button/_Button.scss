.card-category-buttons {
    background-color: $main-button-color-inactive;
    border-radius: 4px;
    color: #31394D;
    font-family: $main-font;
    margin: 20px;
    padding: 10px;
    width: 100px;
        &:hover {
            background-color: $main-button-color-active;
            box-shadow: $active-button-box-shadow;
            cursor: pointer;
            font-weight: 600;
        }
}