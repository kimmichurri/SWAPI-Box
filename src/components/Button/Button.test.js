import React from 'react';
import Button from './Button';
import { shallow } from 'enzyme';

describe('Button', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Button />)
  })

  it ('should match the snapshot', () => {
    expect(wrapper.debug()).toMatchSnapshot()
  })
})
