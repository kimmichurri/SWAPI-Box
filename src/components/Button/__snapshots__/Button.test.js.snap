// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should match the snapshot 1`] = `
"<div>
  <button onClick={[undefined]} className=\\"card-category-buttons people\\">
    people
  </button>
  <button onClick={[undefined]} className=\\"card-category-buttons planets\\">
    planets
  </button>
  <button onClick={[undefined]} className=\\"card-category-buttons vehicles\\">
    vehicles
  </button>
</div>"
`;
